tools/biomodelsML/bio_ml_tool.xml

```diff
@@ -0,0 +1,89 @@
+<tool id="bio_ml_tool" name="Random Forest model" version="0.0.1" profile="22.05">
+    <description>to predict efficacy of immune checkpoint blockade across multiple cancer patient cohorts</description>
+    <requirements>
+        <container type="docker">docker.io/anupkumar/immune-checkpoint:latest</container>
+    </requirements>
+    <command><![CDATA[
+    cp /home/\$NB_USER/forest16.onnx `pwd` && python '$biom_script'
```

Comment:
```suggestion
    cp /home/\$NB_USER/forest16.onnx ./ &&
    python '$biom_script'
```

Reply:
I fixed the permissions in the container, let's see. Not sure if we need to create a new user just like the GPU based container.